sdk/dotnet/Pulumi/Core/OutputUtilities.cs

```diff
@@ -23,6 +23,13 @@ public static async Task<bool> GetIsKnownAsync<T>(Output<T> output)
             return data.IsKnown;
         }
 
+        /// <summary>
+        /// Retrieve the value of the given output.
```

Comment:
We should add a warning similar to the above `GetIsKnownAsync` to discourage people from using it.

Reply:
Done.